crates/bevy_reflect/src/impls/std.rs

```diff
@@ -1039,6 +1039,131 @@ impl FromReflect for Cow<'static, str> {
     }
 }
 
+impl<T: FromReflect + Clone + Send + Sync> Array for Cow<'static, [T]> {
```

Comment:
I don't think the `Clone` bound is necessary (for any of the impls). I'd recommend taking a look at the trait implementations for [`[T; N]`](https://github.com/bevyengine/bevy/blob/f3b8ff65497239a0bf8f208b9c0e089a534d2d3c/crates/bevy_reflect/src/impls/std.rs#L537-L694).

Reply:
I thought Clone would be necessary, because otherwise I get the error: [T]: ToOwned is not satisfied in Cow<'static, [T]>.

[Here](https://doc.rust-lang.org/std/borrow/enum.Cow.html) are the docs on Cow where it mentions the ToOwned bound. I would like to remove the Clone bounds if possible, but that's the only way I've been able to sneak past the type checker so far.

Reply:
Are you sure that the `Send` and `Sync` bounds are necessary?

Reply:
> Are you sure that the `Send` and `Sync` bounds are necessary?

Good point, they should be added by the `FromReflect` bound already.

Reply:
Thanks for pointing that out. I removed them in the latest commit. I think I added them before the ```FromReflect``` bound was there and I didn't realize they were no longer needed.